dist/Safe/t/safeload.t

```diff
@@ -35,3 +35,8 @@ $r or diag $@;
 # perl version in 5.10-.
 ok !$c->reval(q{use 5.012; $undeclared; 1}),
    'reval does not prevent use 5.012 from enabling strict';
+
+# "use Tie::Scalar" depends on UNIVERSAL::import has Tie::Scalar does not have
```

Comment:
I think this was meant to be "as" instead of "has".

Reply:
Fixed, thanks for noticing.